### PR TITLE
[NFTEDA-784, NFT-799]QA Nits

### PR DESCRIPTION
**Pull request by wilsoncusack**
https://github.com/code-423n4/2022-12-backed-findings/issues/292 https://gist.github.com/itsmetechjay/c5eb7f988735188a7bc9005f7b54a9cf

### CODE REVIEW

**Comment by wilsoncusack** [comment]
also https://github.com/code-423n4/2022-12-backed-findings/issues/274

File: src/ReservoirOracleUnderwriter.sol [hunk 1]
```diff
@@ -1,5 +1,5 @@
 // SPDX-License-Identifier: MIT
-pragma solidity ^0.8.17;
+pragma solidity >=0.8.0;
```
Comment (adamgobes) [inline src/ReservoirOracleUnderwriter.sol:2]: shouldn't we use same version for all files?

File: src/UniswapOracleFundingRateController.sol [hunk 2]
```diff
@@ -90,26 +90,27 @@ contract UniswapOracleFundingRateController is IUniswapOracleFundingRateControll
...
-        emit UpdateTarget(target);
+        emit UpdateTarget(_target_);
     }
 
     /// @notice Updates `pool`
     /// @dev reverts if new pool does not have same token0 and token1 as `pool`
     /// @dev if pool = address(0), does NOT check that tokens match papr and underlying
     function _setPool(address _pool) internal {
-        if (pool != address(0) && !UniswapHelpers.poolsHaveSameTokens(pool, _pool)) revert PoolTokensDoNotMatch();
+        address currentPool = pool;
```
Comment (adamgobes) [inline src/UniswapOracleFundingRateController.sol:112]: why the currentPool variable here?

File: src/NFTEDA/NFTEDA.sol [hunk 3]
```diff
@@ -111,7 +111,7 @@ abstract contract NFTEDA is INFTEDA {
     /// @param startTime The start time of the auction
     /// @param auction The auction for which the caller wants to know the current price
     /// @return price the current amount required to purchase the NFT being sold in this auction
-    function _auctionCurrentPrice(uint256 id, uint256 startTime, INFTEDA.Auction memory auction)
+    function _auctionCurrentPrice(uint256, uint256 startTime, INFTEDA.Auction memory auction)
```
[the remark this answers is not in the document]
Comment (wilsoncusack) [reply src/NFTEDA/NFTEDA.sol:114]: we do not use it here...but maybe some readability hit

File: src/ReservoirOracleUnderwriter.sol (hunk 1 above)
Comment (wilsoncusack) [reply src/ReservoirOracleUnderwriter.sol:2]: I think most of these can have an open solc. PaprController is what will compile them all and will force all to what it uses

File: src/UniswapOracleFundingRateController.sol (hunk 2 above)
Comment (wilsoncusack) [reply src/UniswapOracleFundingRateController.sol:112]: to save an SLOAD

**Review by adamgobes — DISMISSED** [review]

**Comment by wilsoncusack** [comment]
https://github.com/code-423n4/2022-12-backed-findings/blob/main/data/Aymen0909-G.md

File: src/NFTEDA/NFTEDA.sol (hunk 3 above)
[the remark this answers is not in the document]
Comment (wilsoncusack) [reply src/NFTEDA/NFTEDA.sol]: reverted this

File: test/paprController/PurchaseLiquidationAuctionNFT.t.sol [hunk 4]
```diff
@@ -296,6 +296,18 @@ contract PurchaseLiquidationAuctionNFT is BasePaprControllerTest {
...
+        nft.mint(borrower, tokenId);
+        vm.stopPrank();
+        vm.startPrank(borrower);
+        nft.approve(address(controller), tokenId);
+        collateral.id = tokenId;
+        IPaprController.Collateral[] memory c = new IPaprController.Collateral[](1);
+        c[0] = collateral;
+        controller.addCollateral(c);
+        vm.stopPrank();
+        //
```
Comment (adamgobes) [inline test/paprController/PurchaseLiquidationAuctionNFT.t.sol:309]: nit stray comment

**Review by adamgobes — APPROVED** [review]

File: test/paprController/PurchaseLiquidationAuctionNFT.t.sol (hunk 4 above)
Comment (wilsoncusack) [reply test/paprController/PurchaseLiquidationAuctionNFT.t.sol:309]: intentional :)

**Pull request merged**